infer/tests/codetoanalyze/c/bufferoverrun/break_continue_return.c

```diff
@@ -0,0 +1,18 @@
+int break_continue_return(){
```

Comment:
please run`clang-format -i <FILE>` on all c files. You can use one from `path-to-infer/facebook-clang-plugins/clang/install/bin`

It will probably change `.exp` file so remember to change it